dateutil/test/test_parser.py

```diff
@@ -877,3 +877,14 @@ def test_validate_hour(self):
         invalid = "201A-01-01T23:58:39.239769+03:00"
         with self.assertRaises(ValueError):
             parse(invalid)
+
+    @unittest.skip("Known failure")
```

Comment:
For the purposes of treating this like a TODO list, I'll follow up to put these into a separate class.

Reply:
Once we have `pytest`, we should refactor these as `@pytest.mark.xfail`, since that acts as a sentinel for any PRs that might accidentally fix it in the future.